indexer/rollout_new_release.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+source args.sh
+
+set -ex
+
+# Start by creating a new image.
+. push_new_docker_image.sh $@
```

Comment:
any reason to source this rather than run it?

Reply:
well, it actually ensures we're printing the correct usage message as this way we preserve $0 (and set -ex) from outer call.